streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -48,16 +46,16 @@ import _root_.scala.collection.JavaConverters._
  */
 class TopologyTest extends JUnitSuite {
 
-  val inputTopic = "input-topic"
-  val userClicksTopic = "user-clicks-topic"
-  val userRegionsTopic = "user-regions-topic"
+  private val inputTopic = "input-topic"
+  private val userClicksTopic = "user-clicks-topic"
+  private val userRegionsTopic = "user-regions-topic"
 
-  val pattern = Pattern.compile("\\W+", Pattern.UNICODE_CHARACTER_CLASS)
+  private val pattern = Pattern.compile("\\W+", Pattern.UNICODE_CHARACTER_CLASS)
 
-  @Test def shouldBuildIdenticalTopologyInJavaNScalaSimple() = {
+  @Test def shouldBuildIdenticalTopologyInJavaNScalaSimple(): Unit = {
 
     // build the Scala topology
-    def getTopologyScala(): TopologyDescription = {
+    def getTopologyScala: TopologyDescription = {
```

Comment:
Scala style is to eschew empty parens when the method doesn't produce side effects. This one does not, so I've removed the parens.